### ISSUE TITLE
Duplicate scripts?

### ISSUE THREAD

**Issue by vanduong28** [ga-wdi-boston/api-token-auth#34]
@payne-chris-r asked me to open this. There are two scripts for the same thing and it's confusing.

change-password-json.sh and change-password.sh

**Comment by gaand** [comment]
There are two script that use different techniques to achieve the same result.  This is important.

**Comment by payne-chris-r** [comment]
Yeah, I know. I'm not sure they still follow the same convention, and I think we might want scripts/json or scripts/?? so people don't run the wrong one from the wrong path?

**Comment by gaand** [comment]
I honestly don't understand what your point is. The files that have `json` in the name are for sending JSON.  The file without send the default (url encoded).
